cmake/version.cmake

```diff
@@ -4,32 +4,31 @@ function(git_version _var)
 		execute_process(COMMAND ${GIT_EXECUTABLE} describe --tags --always
 			WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
 			OUTPUT_VARIABLE out
+			RESULT_VARIABLE ret
+			ERROR_QUIET
 			OUTPUT_STRIP_TRAILING_WHITESPACE)
-			
-		 set(${_var}
-			  "${out}"
-			  PARENT_SCOPE)
+		
+		if(ret AND NOT ret EQUAL 0)
```

Comment:
Is that really necessary to double check that ret is not equal to 0 ?

Reply:
Indeed that check was useless